app/controllers/flights_controller.rb

```diff
@@ -1,7 +1,8 @@
 class FlightsController < ApplicationController
   def index
     @flight = Flight.new
-    @flights = Flight.order("RANDOM()").limit(50)
+    @flights = Flight.order("RANDOM()").
+      paginate(:page => params[:page], :per_page => 30)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Align the operands of an expression in an assignment spanning multiple lines.